src/Telegram/Types/Message.php

```diff
@@ -417,4 +418,16 @@ protected function mapSubObjects(string $key, array $data): TelegramTypes
         // Return always null if none of the objects above matches
         return parent::mapSubObjects($key, $data);
     }
+
+    /**
+     * Decode style entities from Telegram bot messages (bold, italic, etc.) in text with inline entities that duplicate (when possible) the exact style the message had originally when was sended to the bot.
+     *
+     * @param string $style
```

Comment:
As for comments, this is the place to talk about accepted values. 

You can do something like `@param string $style Use 'HTML', 'MarkdownV2' or 'Markdown'`